<launch>	
	<node pkg="laser_node" name="laser_node" type="laser_node"/>
	<node pkg="pkg_odometry" name="node_odometry" type="node_odometry" output="screen" launch-prefix="xterm -e"/>
	<node pkg="pkg_sensors_tf" name="node_sensors_tf" type="node_sensors_tf"/>
	<node pkg="map_server" name="map_server" type="map_server" args="/home/user/testMap.yaml"/>
	<node pkg="amcl" name="amcl" type="amcl">
	    <param name="min_particles" value="500"/>  <!--100-->
	    <param name="max_particles" value="5000"/> <!--5000-->
	    <param name="kld_err" value="0.01"/>       <!--0.01-->
	    <param name="kld_z" value="0.99"/>         <!--0.99-->
	    <param name="update_min_d" value="-1"/>   <!--0.2-->
	    <param name="update_min_a" value="-1"/>  <!--pi/6-->
	    <param name="transform_tolerance" value="0.1"/> <!--0.1-->
	    
	    <param name="laser_min_range" value="-1"/>  <!---1-->
	    <param name="laser_max_range" value="-1"/>  <!---1-->
	    <param name="laser_model_type" value="likelihood_field"/>  <!--likelihood_field-->
	    
	    <param name="odom_model_type" value="diff"/>  <!--diff-->
	    <param name="odom_alpha_1" value="0.2"/>      <!--0.2-->
	    <param name="odom_alpha_5" value="0.2"/>      <!--0.2-->
	</node>
	    
	<node pkg="pkg_pose" name="node_pose" type="node_pose"/>
</launch>
